packages/common/object/src/map-object.ts

```diff
@@ -0,0 +1,16 @@
+import { PlainObjectOf } from '@angular-ru/common/typings';
+
+export function mapObject<Type, ReturnType>(
+    object: PlainObjectOf<Type>,
+    mapper: (obj: Type, key: string) => ReturnType
+): PlainObjectOf<ReturnType> {
+    const result: PlainObjectOf<ReturnType> = {};
```

Comment:
@Schirbak I recommended use JSON.parse with JSON.stringify for mapping fields
analogue - https://github.com/Angular-RU/angular-ru-sdk/blob/master/packages/common/object/src/replace-with-null.ts

Reply:
this function is not for deep mapping